lib/panda_motd/components/fail_2_ban.rb

```diff
@@ -0,0 +1,45 @@
+require 'pry'
```

Comment:
Do we need `pry` here?

Reply:
Dammit! Good catch.

Reply:
It's fine :)
Good that I decided to review this :)

Reply:
Resolved in 667ab0a3226e1ac1f799c67324af50a490157687.